source/get-started/virtual-machine-install/virt-manager.rst

```diff
@@ -0,0 +1,234 @@
+         :alt: Choose storage volume
+
+         Figure 6: Choose storage volume dialog box
+
+   #. In the `Choose the operating system you are installing` search field, 
+      begin typing `generic` until you see the `Generic default` value and 
+      then select that. Once finished on this screen, select the
+      :guilabel:`Forward` as shown in figure 7.
+
+      .. figure:: figures/virtmgr/virt-manager-07.png
+         :scale: 100%
+         :alt: New VM
+
+         Figure 7: New VM dialog box
+
+      .. note::
+
+         You may get a message that the emulator may not have search
+         permissions for the path you entered for the ISO image and it will 
+         want to correct that.  Select :guilabel:`Yes` to proceed to the next
+         step.
+
+#. In step 3 of the `New VM` wizard you set the memory and CPUs to allocate 
+   for your new VM. Depending on the resources available to you on your host
+   system, select what is appropriate. In this example I set `Memory` to
+   2048GB and `CPUs` to 1. Once complete, select the :guilabel:`Forward` as 
+   shown in figure 8, to go to the next step.
+
+   .. figure:: figures/virtmgr/virt-manager-08.png
+      :scale: 100%
+      :alt: New VM Choose Memory and CPU settings dialog box
+
+      Figure 8: New VM Choose Memory and CPU settings dialog box
+
+#. Step 4 of the `New VM` wizard will set up your storage media for your VM.
+   You can create a new disk image or select to use an existing image. For
+   this exercise we select the `Enable storage for this virtual machine` and
+   create a 20GB image to go with. Once set as shown in figure 9, select the
+   :guilabel:`Forward` button.
+
+   .. figure:: figures/virtmgr/virt-manager-09.png
+      :scale: 100%
+      :alt: New VM Enable storage dialog box
+
+      Figure 9: New VM Enable storage dialog box
+
+#. Step 5 of the wizard will show you the selections you have made and give
+   you the ability to customize the configuration before running the
+   installation. Select the `Customize configuration before install` check-box
+   and then select the :guilabel:`Finish` button.
+
+   .. figure:: figures/virtmgr/virt-manager-10.png
+      :scale: 100%
+      :alt: New VM Ready to begin the installation dialog box
+
+      Figure 10: New VM Ready to begin the installation dialog box
+
+#. Clear Linux requires an UEFI firmware to boot and operate correctly so you
+   need to customize the installation process by changing the firmware from
+   `BIOS` to `UEFI x86_64`. In the `Firmware` field, select the
+   `UEFI x86_64: /usr/share/qemu/OVMF.fd` entry as shown in figure 11 and 
+   select the :guilabel:`Apply` button.
+
+   .. figure:: figures/virtmgr/virt-manager-11.png
+      :scale: 100%
+      :alt: vm1 on QEMU/KVM dialog box
+
+      Figure 11: vm1 on QEMU/KVM dialog box
+
+#. Begin the installation by selecting the :guilabel:`Begin Installation` in
+   the upper right hand corner of the `vm1 on QEMU/KVM` dialog box.
```

Comment:
..  in the upper left

Reply:
All good suggestions and they have been made.  I also had a right hand reference for another left hand one that  I fixed as well.  That's what happens when you're a lefty...everything is on the "right" side! :)